### PR TITLE
develop: update tidb cloud import steps

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/10987
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-bookshop-schema-design.md [hunk 1]
```diff
@@ -70,9 +70,10 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
 
 ### 方法二：通过 TiDB Cloud Import 功能
 
-在 TiDB Cloud 的数据库详情页面，你可以通过点击 **Import** 按钮，进入到 **Data Import Task** 页面，在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud：
+在 TiDB Cloud 的集群详情页面，你可以通过切换到 **Import** 标签页，点击 **Import** 按钮进入到 **Data Import** 页面，在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud 集群(详细步骤请参考 [Migrate From Amazon S3 or GCS (English Only)](https://docs.pingcap.com/tidbcloud/migrate-from-amazon-s3-or-gcs#step-3-import-data-into-tidb-cloud) 指南)：
```
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:73]:
```suggestion
在 TiDB Cloud 的集群详情页面，你可以通过切换到 **Import** 标签页，点击 **Import** 按钮进入到 **Data Import** 页面，在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud 集群：
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 2]
```diff
@@ -90,24 +91,15 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
...
-
-   如果导入过程中出现如下错误信息，你需要通过 `DROP DATABASE bookshop;` 命令将原来创建的示例数据库进行清除后再重新导入。
-
-   > table(s) [`bookshop`.`authors`, `bookshop`.`book_authors`, `bookshop`.`books`, `bookshop`.`orders`, `bookshop`.`ratings`, `bookshop`.`users`] are not empty.
+    在这个示例数据当中，预先生成了 20 万的用户信息、50 万条书籍信息、10 万条作者信息、100 万条评分记录以及 100 万条订单信息。
+   
+5. 点击 **Start Import** 按钮开始导入数据，等待 TiDB Cloud 完成数据导入。
 
-7. 你可以通过 [TiDB Cloud 文档](https://docs.pingcap.com/tidbcloud)获取更多有关 TiDB Cloud 的信息。
+你可以通过 [TiDB Cloud 文档](https://docs.pingcap.com/tidbcloud)获取更多有关 TiDB Cloud 的信息。
```
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:102]:
```suggestion
你可以通过 [TiDB Cloud Migration Overview](https://docs.pingcap.com/tidbcloud/tidb-cloud-migration-overview) 文档获取更多有关数据导入和迁移的信息。
```

**Review by qiancai — APPROVED** [review]

File: develop/dev-guide-bookshop-schema-design.md [hunk 3]
```diff
@@ -70,9 +70,10 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
 
 ### 方法二：通过 TiDB Cloud Import 功能
 
-在 TiDB Cloud 的数据库详情页面，你可以通过点击 **Import** 按钮，进入到 **Data Import Task** 页面，在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud：
+在 TiDB Cloud 的集群详情页面，你可以通过切换到 **Import** 标签页，点击 **Import** 按钮进入到 **Data Import** 页面，在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud 集群：
```
Comment (qiancai) [inline develop/dev-guide-bookshop-schema-design.md:73]:
```suggestion
在 TiDB Cloud 的集群详情页面，你可以通过切换到 **Import** 标签页，点击 **Import** 按钮进入到 **Data Import** 页面。在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud 集群：
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 4]
```diff
@@ -70,9 +70,10 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
 
 ### 方法二：通过 TiDB Cloud Import 功能
 
-在 TiDB Cloud 的数据库详情页面，你可以通过点击 **Import** 按钮，进入到 **Data Import Task** 页面，在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud：
+在 TiDB Cloud 的集群详情页面，你可以通过切换到 **Import** 标签页，点击 **Import** 按钮进入到 **Data Import** 页面。在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud 集群：
```
Comment (Oreoxmt) [inline develop/dev-guide-bookshop-schema-design.md:73]:
```suggestion
在 TiDB Cloud 的集群详情页面，你可以通过切换到 **Import** 标签页，点击 **Import Data** 按钮进入到 **Data Import** 页面。在该页面当中，按照以下步骤将 Bookshop 示例数据从 AWS S3 中导入到你的 TiDB Cloud 集群：
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 5]
```diff
@@ -70,9 +70,10 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
...
 
-1. 将以下 **Bucket URL** 和 **Role-ARN** 复制到页面上对应的输入框当中：
+1. 选择 **Data Format** 为 **SQL File**。
+2. 将以下 **Bucket URL** 和 **Role-ARN** 复制到页面上对应的输入框当中：
```
Comment (Oreoxmt) [inline develop/dev-guide-bookshop-schema-design.md:76]:
```suggestion
2. 将以下 **Bucket URI** 和 **Role-ARN** 复制到页面上对应的输入框当中：
```
Role-ARN is not in the Data Import page

File: develop/dev-guide-bookshop-schema-design.md [hunk 6]
```diff
@@ -70,9 +70,10 @@ tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --rati
...
 
     **Bucket URL**:
```
Comment (Oreoxmt) [inline develop/dev-guide-bookshop-schema-design.md:78]:
```suggestion
    **Bucket URI**:
```

Comment (Oreoxmt) [reply develop/dev-guide-bookshop-schema-design.md:78]: L86: `**Role ARN**`

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**